src/jsonSchemas/mapContributorConf.schema.json

```diff
@@ -14,6 +14,14 @@
             "description": "Id of entity to show on Map",
             "type": "string"
         },
+        "includeFeaturesFields": {
+            "description": "List of fields names that will be included in features mode as geojson properties.",
```

Comment:
List of fields pattern or names that will be included in features mode as geojson properties.